Sources/Composable.swift

```diff
@@ -98,6 +98,19 @@ public extension NSAttributedString {
         return string
     }
 
+    public func styled(with style: StringStyle, _ overrideParts: StringStyle.Part...) -> NSAttributedString {
+        let newStyle = style.byAdding(overrideParts)
+        let newAttributes = newStyle.attributes
+
+        let mutableSelf = mutableStringCopy()
```

Comment:
Wondering if this could be simplified to something like this?

```swift
public func styled(with style: StringStyle, _ overrideParts: StringStyle.Part...) -> NSAttributedString {
    return string.styled(with: style, overrideParts)
}
```

And then changing up the `Composable` extension to something like this
```swift
    public func styled(with style: StringStyle, _ overrideParts: StringStyle.Part...) -> NSAttributedString {
        return styled(with: style, overrideParts)
    }

    public func styled(with style: StringStyle, _ overrideParts: [StringStyle.Part]) -> NSAttributedString {
        let string = NSMutableAttributedString()
        let newStyle = style.byAdding(stringStyle: StringStyle(overrideParts))
        append(to: string, baseStyle: newStyle)
        return string
    }
```

Or is there other data on the `NSAttributedString` we're concerned about copying over besides attributes?

Reply:
This causes the new test introduced in this PR to fail. I don't remember in great detail what's going on in that case, but the method names in this area are kind of borked, and it was an unrelated yak shave to try to clean them up. This method needs to behave fundamentally differently from the others that look like it, in terms of what's being inherited and what's being overridden, but it's hard to articulate why because it takes a while for me just to load the meanings of each of the poorly-named methods into my mental RAM 😩. Definitely good fodder for future cleanup, but I really wanted to get this bug fixed quickly, and the tests give me confidence that, bad naming be damned, this is ready to go.